lib/constants.js

```diff
@@ -0,0 +1,13 @@
+'use babel'
+
+export const MachineStatusText = {
+  1: 'Offline',
+  2: 'Online',
+  3: 'Connected'
+}
+
+export const MachineStatus = {
+  Offline: 1,
+  Online: 2,
+  Connected: 3
+}
```

Comment:
One array of `[ null, 'Offline', 'Online', 'Connected']` would have been enough for both uses imo.